python/ray/serve/_private/common.py

```diff
@@ -33,6 +33,8 @@ class ApplicationStatus(str, Enum):
     DEPLOYING = "DEPLOYING"
```

Comment:
What's the difference between `NOT_STARTED` and `NOT_EXISTED`? Do we need both?

Reply:
NOT_EXISTED is used when client is getting a non-existed application.